component/s3storage/s3storage_test.go

```diff
@@ -2724,6 +2701,7 @@ func (s *s3StorageTestSuite) TestFlushFileUpdateChunkedFile() {
 	blockSizeMB := 5
 	blockSizeBytes := blockSizeMB * common.MbToBytes
 	storageTestConfigurationParameters.PartSizeMb = int64(blockSizeMB)
+	storageTestConfigurationParameters.UploadCutoffMb = 5
```

Comment:
Does this mean it wasn't actually testing the uploader before?
Nice catch!

Reply:
Yes, I guess so, since the tests weren't passing until I added this.